### PR TITLE
feat: add upload test result

### PR DESCRIPTION
**Pull request by LiteSun**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [ ] New feature provided
- [x] Improve performance
- [ ] Backport patches
___
### New feature or improvement
- Describe the details and related test reports.
Currently front-end e2e is running very unstable on GitHub action, add upload files to github and download them for developers to locate the problem

The output file is here:
image

### CODE REVIEW

**Review by juzhiyuan — APPROVED** [review]
LGTM, and please add `How to get report` guide.

**Review by nic-chen — APPROVED** [review]

**Review by starsz — APPROVED** [review]
Pretty good!

**Comment by juzhiyuan** [comment]
@LiteSun I couldn't find that file.

image

File: .github/workflows/frontend-e2e-test.yml [hunk 1]
```diff
@@ -57,3 +57,10 @@ jobs:
       - name: Upload coverage to Codecov
         run: |
          bash <(curl -s https://codecov.io/bash) -f ./coverage/coverage-final.json -F frontend-e2e-test
+
+      - name: Archive code coverage results
+        uses: actions/upload-artifact@v2
+        if: always()
+        with:
+          name: cypress-report
+          path: web/cypress/
```
Comment (starsz) [inline .github/workflows/frontend-e2e-test.yml:66]: Need to set a short expire time.

**Review by starsz — CHANGES_REQUESTED** [review]

**Review by gxthrj — APPROVED** [review]

File: .github/workflows/frontend-e2e-test.yml (hunk 1 above)
Comment (LiteSun) [reply .github/workflows/frontend-e2e-test.yml:66]: updated, thanks.

**Review by starsz — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Pull request merged**